message_ix_models/model/material/build.py

```diff
@@ -169,16 +129,31 @@ def build(
         last_hist_year = scenario.par("historical_activity")["year_act"].max()
         modify_industry_demand(scenario, last_hist_year, iea_data_path)
         add_new_ind_hist_act(scenario, [last_hist_year], iea_data_path)
-        add_elec_i_ini_act(scenario)
         add_emission_accounting(scenario)
 
-        # scenario.commit("no changes")
+    if modify_existing_constraints:
```

Comment:
Same as above: is is feasible to write a test that calls this function once with `modify_existing_constraints = True` and once with `modify_existing_constraints = False`?

Reply:
We have the same problem here as explained in this [comment](https://github.com/iiasa/message-ix-models/pull/218#discussion_r1796837000) on another suggestions. Testing would either require a customized bare RES or creating a specific MESSAGEix-GLOBIOM snapshot. I know that @GamzeUnlu95 is working on an updated R12 snapshot that is supposed to be added on Zenodo, but not sure when this will be finalized.